src/hooks/useWebsocket/index.ts

```diff
@@ -0,0 +1 @@
+export * from './useWebsocket.ts'
```

Comment:
I would remove this file and put `useWebsockets.ts` like this, without the additional folder:
```
hooks
|-- useWebsockets.ts
```
because I think there will be no other files inside this directory.

The index.ts file is useful in components directories, but with util functions, hooks, etc., it makes less sense.

Reply:
Done